src/Hedgehog/Property.fs

```diff
@@ -81,11 +81,32 @@ module Property =
     let bind (k : 'a -> Property<'b>) (m : Property<'a>) : Property<'b> =
         bindGen (toGen << k) (toGen m) |> ofGen
 
+#if !FABLE_COMPILER
+    let private (|IsGenericList|_|) (candidate : obj) : seq<obj> option =
```

Comment:
What about renaming this to `IsResizeArray`?

Reply:
Oh, yes.  That makes sense.  Otherwise it would suggest that adding any instance of type `obj` were possible, which is not the case.

Reply:
Done